From 70e77dd3c8f9cadf3a73bd662086bdaa3f482886 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 10 Jul 2023 19:12:30 -0600
Subject: [PATCH] Bump Security String to 2023-07-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-20918  A-243794108    EoP    High       11, 12, 12L, 13
CVE-2023-21238  A-277740848    ID     High       11, 12, 12L, 13
CVE-2023-21240  A-275340417    DoS    High       11, 12, 12L, 13
CVE-2023-21241  A-271849189    EoP    High       11, 12, 12L, 13
CVE-2023-21243  A-274445194    DoS    High       11, 12, 12L, 13
CVE-2023-21245  A-222446076    EoP    High       11, 12, 12L, 13
CVE-2023-21246  A-273729476    EoP    High       11, 12, 12L, 13
CVE-2023-21250  A-261068592    RCE    Critical   11, 12, 12L, 13
CVE-2023-21251  A-204554636    EoP    High       11, 12, 12L, 13
CVE-2023-21261  A-271680254    ID     High       11, 12, 12L, 13

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2023-21087  A-261723753    DoS    High       11, 12, 12L, 13         62fa996fb636
CVE-2023-21145  A-265293293    EoP    High       11, 12, 12L, 13         5fb1e62ea019

Not Implemented:
=======================
None

Not Applicable (platform source):
=================================
Remote History
WAPI
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-2136   A-278113033    RCE    High       13
CVE-2023-20910  A-245299920    DoS    High       11, 12, 12L, 13
CVE-2023-20942  A-258021433    EoP    High       12, 12L, 13
CVE-2023-21239  A-274592467    ID     High       12, 12L, 13
CVE-2023-21247  A-277333781    EoP    High       12, 12L, 13
CVE-2023-21248  A-277333746    EoP    High       12, 12L, 13
CVE-2023-21249  A-217981062    ID     High       13
CVE-2023-21254  A-254736794    EoP    High       13
CVE-2023-21256  A-268193384    EoP    High       13
CVE-2023-21257  A-257443065    EoP    High       13
CVE-2023-21262  A-279905816    EoP    High       12, 12L, 13

Change-Id: I70063f7ffbe701377a397a56d943a02798b123a6
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 9da0446807..183a223f2b 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2023-06-05
+      PLATFORM_SECURITY_PATCH := 2023-07-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.40.1

